apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -59,6 +59,16 @@ const Dashboard = () => {
     }
   };
 
+  const isValidAddress = useCallback(
+    (address: string | null) => {
+      if (!address) {
+        return false;
+      }
+      return services?.web3?.utils.isAddress(address);
```

Comment:
We could simply use `Web3.utils.isAddress` but since we already depend on `services`, it's fine.